ReactNativeSearchBar.podspec

```diff
@@ -0,0 +1,17 @@
+require "json"
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+
+Pod::Spec.new do |s|
+  s.name          = "ReactNativeSearchBar"
+  s.version       = package["version"]
+  s.source_files  = "**/*.{h,m}"
```

Comment:
This pattern matches the sample code. Please update it to `"RNSearchBar*.{h,m}"` or perhaps `"RNSearchBar.{h,m}", "RNSearchBarManager.{h,m}"`